src/app/customizer/GitHub-Profile-generator/page.jsx

```diff
@@ -0,0 +1,1052 @@
+                  {skills.map((skill, index) => (
+                    <SkillIcon key={index} skill={skill} />
+                  ))}
+                </div>
+              </div>
+            )}
+
+            {/* GitHub Stats */}
+            {formValues.githubUsername && (
+              <div className="mt-4">
+                <h1 className="text-2xl font-bold mt-8 mb-4">
+                  📊 GitHub Stats
+                </h1>
+                <div className="flex flex-col items-center">
+                  <img
+                    src={`http://github-readme-streak-stats.herokuapp.com?user=${formValues.githubUsername}&theme=blue-green`}
+                    alt="GitHub Streak"
+                    className="mb-4"
+                  />
+                  <img
+                    src={`https://bad-apple-github-readme.vercel.app/api?username=${formValues.githubUsername}&show_icons=true&count_private=true&line_height=20&icon_color=00b3ff&theme=blue-green&title_color=00b3ff`}
+                    alt="GitHub Stats"
+                    className="mb-4"
+                  />
+                  <img
+                    src={`https://github-readme-stats.vercel.app/api/top-langs/?username=${formValues.githubUsername}&layout=compact&theme=blue-green`}
+                    alt="GitHub Langs"
+                  />
+                </div>
+              </div>
+            )}
+
+            {/* Support Me */}
+            {(formValues.buyMeACoffee || formValues.kofi) && (
+              <div className="mt-4">
+                <h1 className="text-2xl font-bold mt-8 mb-4">💖Support Me!</h1>
+                <div className="flex justify-center space-x-4">
+                  {formValues.buyMeACoffee && (
+                    <a
+                      href={formValues.buyMeACoffee}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <img
+                        src="https://img.shields.io/badge/Buy_Me_A_Coffee-FBBC05?style=for-the-badge&logo=buy-me-a-coffee&logoColor=white"
+                        alt="Buy Me a Coffee Badge"
+                        className="h-6"
+                      />
+                    </a>
+                  )}
+
+                  {formValues.kofi && (
+                    <a
+                      href={formValues.kofi}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <img
+                        src="https://img.shields.io/badge/Ko--fi-F16061?style=for-the-badge&logo=kofi&logoColor=white"
+                        alt="Ko-fi Badge"
+                        className="h-6"
+                      />
+                    </a>
+                  )}
+                </div>
+              </div>
+            )}
+          </div>
+        ) : (
+          <div className="mt-6 p-4 bg-gray-700 rounded-lg">
```

Comment:
```suggestion
          <div className="mt-6 p-4 bg-gray-700 rounded-lg text-white">
```
You need to specify the color.